src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -318,88 +369,20 @@ def forward(
         layer_past: Optional[Tuple[torch.Tensor]] = None,
         use_cache: Optional[bool] = False,
         output_attentions: Optional[bool] = False,
+        padding_mask: Optional[torch.Tensor] = None,
```

Comment:
padding_mask is never used in any attn implementation but to keep things common between the different attns, I added it to the method signature.